pipeline-runner/tests/testthat/test-gem2s-2-load_cellranger.R

```diff
@@ -279,6 +279,36 @@ test_that("load_cellranger loads multisample experiments", {
   unlink(sample_dirs, recursive = TRUE)
 })
 
+test_that("load_user_files reads rhapsody files", {
```

Comment:
what about adding tests for the following cases:
- when the files passed are neither `rhapsody` nor `10x`
- when the passed files are invalid (for example, for rhapsody the received input is not an expression matrix. Or some of the parameters in that matrix are badly formatted or missing)
-